eventhubs/client/pom.xml

```diff
@@ -0,0 +1,93 @@
+<!-- Copyright (c) Microsoft Corporation. All rights reserved.
+     Licensed under the MIT License. -->
+<project xmlns="http://maven.apache.org/POM/4.0.0"
+         xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+         xsi:schemaLocation="http://maven.apache.org/POM/4.0.0
+                             http://maven.apache.org/xsd/maven-4.0.0.xsd">
+  <modelVersion>4.0.0</modelVersion>
+  <parent>
+    <groupId>com.azure</groupId>
+    <artifactId>azure-client-sdk-parent</artifactId>
+    <version>1.0.0</version>
+    <relativePath>../../pom.client.xml</relativePath>
+  </parent>
+
+  <groupId>com.azure</groupId>
+  <artifactId>azure-eventhubs-clients</artifactId>
```

Comment:
Do we need the `-clients`?

Reply:
I wasn't sure because we'd probably have azure-eventhubs, azure-eventhubs-eph? So I wasn't sure if we should have this parent pom just as -clients.

Reply:
Call it -parent